src/pyspiffe/utils/certificate_utils.py

```diff
@@ -83,40 +101,42 @@ def load_certificates_bytes_from_file(certificates_file_path: str) -> bytes:
     """
 
     try:
-        with open(certificates_file_path, 'rb') as certs_file:
-            return certs_file.read()
+        return _load_bytes_from_file(certificates_file_path)
     except FileNotFoundError:
-        raise X509CertificateError(
+        raise LoadCertificateError(
             'Certificates file not found: {}'.format(certificates_file_path)
         )
     except Exception as err:
-        raise X509CertificateError(
+        raise LoadCertificateError(
             'Certificates file could not be read: {}'.format(str(err))
         )
 
 
-def write_certificate_to_file(
-    certificate: Certificate,
-    dest_file: BinaryIO,
+def write_certificates_to_file(
+    certs_file_path: str,
     encoding: serialization.Encoding,
+    certificates: Iterable[Certificate],
 ) -> None:
-    """Writes a certificate to a file.
+    """Writes certificates to a file.
 
     Args:
-        certificate: Certificate object to be saved to file.
-        dest_file: BinaryIO object representing the file to be written to.
-        encoding: The serialization format to use to save the certificate.
+        certs_file_path: Path to the file containing the certificates.
+        encoding: The serialization format to use to save the certificates.
```

Comment:
nit: "The serialization format used to encode the certificates. Can be 'PEM' or 'DER'. "